tests/Doubles/FakeDownloader.php

```diff
@@ -18,8 +18,9 @@ final class FakeDownloader implements Downloader
      */
     private array $content = [];
 
-    public function __construct(string $basePath = __DIR__.'/../Resources')
+    public function __construct(?string $basePath = null)
     {
+        $basePath = $basePath ?? dirname(__DIR__, 1).DIRECTORY_SEPARATOR.'Resources';
```

Comment:
what's the reason for level 1 in dirname and use of DIRECTORY_SEPARATOR?

Reply:
Was just to avoid the relative paths with `../` on it. I had problems when using those in conjunction with the composer repositories, specially the paths. Regarding the `DIRECTORY_SEPARATOR`, I try to use it whenever I can for portability. If you guys prefer just `/` I can replace it as well. 

Now running the suite again with the old code I see that I don't get any errors :man_shrugging: . So I can easily revert it, just say the word. 😃

Reply:
It's ok, it's just that `1` is a default level for the `dirname` function and we don't really support windows in other places 😅

Reply:
OMG, I didn't even see that (default value). I will go back to the old code to avoid confusion.

Reply:
There you go! Thanks for catching such fine grained detail :+1:

Reply:
Cheers, good job overall 🍻